### PR TITLE
Adds trFunc_travelInfectionRate_ageAdjusted & its unit tests. 

### PR DESCRIPTION
**Pull request by dinskid**
Adds trFunc_travelInfectionRate_ageAdjusted.
There is some error with one(or more) unit tests that are grouped under not working.

### CODE REVIEW

File: julia/diseaseProg.jl [hunk 1]
```diff
@@ -2,6 +2,8 @@
 using DataFrames
 using CSVFiles
 using LinearAlgebra
+import StatsFuns.logistic, StatsFuns.gammapdf
```
Comment (aa25desh) [inline julia/diseaseProg.jl:5]:
```suggestion
import StatsFuns: logistic, gammapdf
```

File: julia/diseaseProg.jl [hunk 2]
```diff
@@ -426,3 +428,37 @@ function _einsum3(a, b) #'ijkl,j->i'
     end
     return sum(p, dims=1)
 end
+
+function trFunc_travelInfectionRate_ageAdjusted(
+	t::Int64, # Time within simulation
+
```
Comment (aa25desh) [inline julia/diseaseProg.jl:434]:
```suggestion
```

File: julia/diseaseProg.jl [hunk 3]
```diff
@@ -426,3 +428,37 @@ function _einsum3(a, b) #'ijkl,j->i'
...
+	travelMaxTime::Int64 = 200,
+	travelBaseRate::Float64 = 5e-4, # How many people normally travel back to the country per day # TODO - get data
+	travelDecline_mean::Float64 = 15.,
+	travelDecline_slope::Float64 = 1.,
+
```
Comment (aa25desh) [inline julia/diseaseProg.jl:439]:
```suggestion
```

Comment (aa25desh) [reply julia/diseaseProg.jl:439]: Blank spaces should be avoided and you can write `1.0` instead of `1.` which looks better:)

File: test.jl [hunk 4]
```diff
@@ -19,6 +20,11 @@ initial_state = (nAge=9, nHS=8, nIso=4, nI=4, nTest=4)
 	trFunc_HospitalAdmission(;initial_state...)
 	@test py"np.transpose(trFunc_HospitalDischarge())"≈
 	trFunc_HospitalDischarge(;initial_state...)
+	@test py"np.transpose(trFunc_travelInfectionRate_ageAdjusted(10))"≈
+	trFunc_travelInfectionRate_ageAdjusted(t)
+end
+
+@testset "not working"
```
Comment (aa25desh) [inline test.jl:27]: There is some issue with relative path as we make pkg , things will get resoleved:)

File: julia/diseaseProg.jl (hunk 3 above)
Comment (dinskid) [reply julia/diseaseProg.jl:439]: Okay..I will change it

File: test.jl (hunk 4 above)
Comment (dinskid) [reply test.jl:27]: Okay

**Comment by aa25desh** [comment]
Thanks @dinskid!

**Pull request merged**